data/i18n/en/en.toml

```diff
@@ -1,5 +1,8 @@
 # i18n strings for the English (main) site.
 # NOTE: Please keep the entries in alphabetical order when editing
+[and]
+other = "and"
+
```

Comment:
Thank you for this suggestion, @seokho-son. I'll update this PR accordingly before tomorrow.

Reply:
@rolfedh and @seokho-son, since this group of variables builds a sentence, do you think it would be helpful to add a comment to show how these words are combined together on the page? I think that would help give context, but maybe doesn't solve the problem of allowing folks to use the correct "and" for their language.

Are these localized words used in different places on the site?

Reply:
Hi @a-mccarthy :)

> do you think it would be helpful to add a comment to show how these words are combined together on the page?

I think it is a nice idea. (I usually find shortcodes and related pages by searching variable name for the `T` i18n string)
If there is a comment, l10n would be much easier.

> Are these localized words used in different places on the site?

It depends on layouts/shortcodes that utilize i18n strings.
This PR only uses `[and]` in `layouts/shortcodes/release-data.html` only. But if we make `[and]` is a common i18n string, folks will try to utilize it in any other pages. (maybe it is just my concern)

:)